client/client.go

```diff
@@ -686,10 +685,7 @@ func (r *NotaryRepository) publish(cl changelist.Changelist) error {
 		return err
 	}
 
-	remote, err := getRemoteStore(r.baseURL, r.gun, r.roundTrip)
-	if err != nil {
-		return err
-	}
+	remote := r.remoteStore
```

Comment:
good catch @cyli: I agree we need to wrap the `OfflineStore` and also have a check for `nil` here